.github/workflows/migrations-check.yml

```diff
@@ -18,7 +18,7 @@ jobs:
 
     steps:
     - name: Checkout repo
-      uses: actions/checkout@v1
+      uses: actions/checkout@v2
```

Comment:
we initially thought that merge commits failing on `master` was caused by `checkout` action but that wasn't the issue. further `checkout@v1` take ~1 min to checkout and `v2` takes ~4 seconds since we don't yet have `coverage` issue in edx-platform I think we should switch back to `v2`
I'm not exactly sure but I can look into it if the coverage issue is resolved or not.